.travis.yml

```diff
@@ -64,6 +64,24 @@ matrix:
     env: BEAKER_set="ubuntu-1804"
     bundler_args:
     script: bundle exec rake beaker
+  - rvm: 2.4.2
+    sudo: required
+    services: docker
+    env: BEAKER_set="amazonlinux-2"
+    bundler_args:
+    script: bundle exec rake beaker
+  - rvm: 2.4.2
+    sudo: required
+    services: docker
+    env: BEAKER_set="amazonlinux-201803"
```

Comment:
amazon linux 2018 is missing from vagrant

Reply:
I couldn't find a vagrant box for 2018.

Reply:
It seems Amazon focuses most their effort on official docker containers.

Reply:
ok. we don't need to roll our own